api/thirdpartyresource/helpers.go

```diff
@@ -0,0 +1,133 @@
+package thirdpartyresource
+
+import (
+	"fmt"
+	"github.com/docker/swarmkit/api"
+	"strings"
```

Comment:
We follow the convention of putting standard library imports first, then a blank line, then other imports.

```go
"fmt"
"strings"

"github.com/docker/swarmkit/api""
```